integration/cleanup.go

```diff
@@ -19,29 +19,37 @@ package integration
 import (
 	"fmt"
 	"os/exec"
-	"testing"
 
 	integration_util "github.com/grafeas/kritis/pkg/kritis/integration_util"
 	"k8s.io/api/core/v1"
 )
 
-func cleanupKritis(t *testing.T, ns *v1.Namespace) {
-	deleteObject(t, "validatingwebhookconfiguration",
-		fmt.Sprintf("kritis-validation-hook-%s", ns.Name), nil)
-	deleteObject(t, "validatingwebhookconfiguration",
-		fmt.Sprintf("kritis-validation-hook-deployments-%s", ns.Name), nil)
-	deleteObject(t, "csr",
-		fmt.Sprintf("tls-webhook-secret-cert-%s", ns.Name), nil)
-	deleteObject(t, "secret", aaSecret, ns)
+func cleanupInstall(ns *v1.Namespace) error {
+	var resources = []struct {
+		Type string
+		Name string
+	}{
+		{"validatingwebhookconfiguration", fmt.Sprintf("kritis-validation-hook-%s", ns.Name)},
+		{"validatingwebhookconfiguration", fmt.Sprintf("kritis-validation-hook-deployments-%s", ns.Name)},
+		{"csr", fmt.Sprintf("tls-webhook-secret-cert-%s", ns.Name)},
+		{"secret", aaSecret},
+	}
+	for _, r := range resources {
+		if err := deleteObject(r.Type, r.Name, ns); err != nil {
```

Comment:
The first 3 resources  in the list are installed in default namespace and only `secret` is installed in test namespace

Reply:
Great catch. Fixed this, and the fact that the code was not checking the error code from cleanupInstall.